Description: Fix Clean.
 Clean misses:
    applets/taskmanager/labelled-separator.c => $(VALA_GENERATED_FILES)
    applets/taskmanager/labelled-separator.h => $(VALA_GENERATED_FILES)
    doc/reference/libawn-overrides.txt => Missing empty file. Could add touch doc/reference/libawn-overrides.txt
 It also removes:
    INSTALL => autogenerated
    po/avant-window-navigator.pot
Author: Miguel A. Colón Vélez <debian.micove@gmail.com>
Forwarded: no

--- a/applets/taskmanager/Makefile.am
+++ b/applets/taskmanager/Makefile.am
@@ -127,4 +127,4 @@
 	labelled-separator.vala \
 	$(NULL)
 
-CLEANFILES += $(schema_DATA) $(schemas_DATA) $(BUILT_SOURCES)
+CLEANFILES += $(schema_DATA) $(schemas_DATA) $(BUILT_SOURCES) $(VALA_GENERATED_FILES)
--- a/doc/reference/Makefile.am
+++ b/doc/reference/Makefile.am
@@ -106,6 +106,7 @@
 # This includes the standard gtk-doc make rules, copied by gtkdocize.
 if ENABLE_GTK_DOC
 include $(top_srcdir)/gtk-doc.make
+CLEANFILES += $(DOC_MODULE)-overrides.txt
 endif
 
 if MAINTAINER_MODE
